Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -72,9 +132,23 @@ - (void)commontInit
     [self.bubbleImageView addSubview:self.textLabel];
     OWSAssert(self.textLabel.superview);
 
+    self.footerLabel = [UILabel new];
+    self.footerLabel.font = [UIFont ows_regularFontWithSize:12.f];
+    self.footerLabel.textColor = [UIColor lightGrayColor];
+    [self.footerView addSubview:self.footerLabel];
+
     // Hide these views by default.
     self.bubbleImageView.hidden = YES;
     self.textLabel.hidden = YES;
+    self.dateHeaderLabel.hidden = YES;
+    self.footerLabel.hidden = YES;
+
+    [self.dateHeaderLabel autoPinEdgeToSuperviewEdge:ALEdgeTop];
```

Comment:
You'll find the other constraints elsewhere in this class.  I've moved this constraint there to avoid confusion.  I was trying to separate the transient constraints from the static ones.